dropwizard-metrics-graphite/src/main/java/io/dropwizard/metrics/graphite/GraphiteReporterFactory.java

```diff
@@ -32,7 +32,7 @@
  *     </tr>
  *     <tr>
  *         <td>port</td>
- *         <td>8080</td>
+ *         <td>2003</td>
```

Comment:
I don't know why these were defaulting to `8080`, but I updated them to the standard graphite port of `2003`.